shared/profile/edit-avatar/container.tsx

```diff
@@ -69,32 +69,25 @@ export default Container.connect(
       image: stateProps.image,
       onBack: dispatchProps.onBack,
       onClose: dispatchProps.onClose,
-      onSave: (
-        filename: string,
-        crop?: RPCTypes.ImageCropRect,
-        scaledWidth?: number,
-        offsetLeft?: number,
-        offsetTop?: number
-      ) => {
-        if (
-          wizard &&
-          !!crop &&
-          scaledWidth != undefined &&
-          offsetLeft != undefined &&
-          offsetTop != undefined
-        ) {
-          dispatchProps.onSaveWizardAvatar(filename, {crop, offsetLeft, offsetTop, scaledWidth})
+      onSave: (filename: string, crop?: RPCTypes.ImageCropRect) => {
+        if (wizard) {
```

Comment:
don't even include this case here, assume we won't call `onSave` if we are in the wizard

Reply:
I think onSave for mobile wizard would go here, since it doesn't need the cropped avatar? Should I add a check for if it's `wizard && isMobile`?

Reply:
In that case, you still want to keep the crop so we can eventually send it to the server. Perhaps there are too many cases and I was wrong to suggest splitting it in two functions though.

Reply:
ok I'll change it back to one onSave